src/Config/FileSystemRuntimeConfigLoader.cs

```diff
@@ -104,7 +132,10 @@ public string GetConfigFileName()
         {
             Console.WriteLine($"Loading config file from {path}.");
             string json = _fileSystem.File.ReadAllText(path);
-            return TryParseConfig(json, out config, connectionString: _connectionString, replaceEnvVar: replaceEnvVar);
+            bool res = TryParseConfig(json, out RuntimeConfig, connectionString: _connectionString, replaceEnvVar: replaceEnvVar);
```

Comment:
Not something changed in this PR: In the TryParseConfig() method, the order of parameters is messed up. Could you please fix that?

Reply:
The function below? Seems correct to me.

![image](https://github.com/Azure/data-api-builder/assets/93220300/68662b09-fc6f-4032-9793-277ef6a57ad3)

Reply:
perhaps Ayush meant that most other methods in our project have the `out` parameter as the last parameter in the method signature.